ow_sim_tests/scripts/test_task_deliver_sample.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import roslib
+import unittest
+import owl_msgs.msg
+
+from action_testing import *
+from geometry_msgs.msg import Pose, Quaternion
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'task_deliver_sample'
+roslib.load_manifest(PKG)
+
+class TestTaskDeliverSample(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    rospy.init_node("test_task_deliver_sample")
+
+    set_ignore_action_checks(True)
+    # proceed with test only when ros clock has been initialized
+    while rospy.get_time() == 0:
+      rospy.sleep(0.1)
+
+  def test_01_task_deliver_sample(self):
```

Comment:
I find it odd that a deliver is attempted from the stowed position -- this would never occur in practice.  Is this test useful, @AstroStucky ?

Reply:
Some of the bugs I found were found from calling actions immediately at simulator start-up, so I asked Hao to always call the action being tested at the very start of its test. Basically it has been handy at uncovering simulation start-up timing errors, like a topic being subscribed to before it is published. Fixing these can only make our code more rigorous.